VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```

Comment:
[CODE_SMELL] Remove the unused private field 'ChangePassword'.

Reply:
/workitem

Reply:
WorkItem 768 Created see [workItem](Microsoft.VisualStudio.Services.WebApi.ReferenceLink)